examples/cis1-multi/src/lib.rs

```diff
@@ -382,26 +394,21 @@ fn contract_update_operator<A: HasActions>(
     Ok(A::accept())
 }
 
-/// Get the balance of a given token id for a given address and callback
-/// contract function on sender with the result.
-/// Will only succeed if called a smart contracts.
+#[allow(dead_code)]
+type ContractBalanceOfQueryParams = BalanceOfQueryParams<ContractTokenId>;
```

Comment:
Why is the allow(dead_code) needed? If it is only needed for the schema it would still be better to make it `pub` and avoid the need for `allow(dead_code)`.

Reply:
Also please document the type.